catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/download-manager.xml

```diff
@@ -18,7 +18,7 @@
 
     <bean id="downloadThreadPool" class="java.util.concurrent.Executors"
           factory-method="newFixedThreadPool">
-        <argument value="$[org.codice.ddf.system.threadPoolSize]"/>
+        <argument value="${org.codice.ddf.system.threadPoolSize}"/>
```

Comment:
❗️ Don't think this will work since this namespace the place holder https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L25 .

Reply:
It actually broke all these ones when i changed the property placeholder in that main blueprint file 😄 
It apparently inherits that setting in all these other blueprints

Reply:
Sorry, didn't see it was changed 🤦‍♂️ .